back-end/config/config.js

```diff
@@ -0,0 +1,14 @@
+const { Pool } = require('mysql');
+
+var databaseOptions = {
+    host     : 'localhost',
+    database : 'test',
+    user     : 'root',
+    password : 'root',
+    port     : '8889'
+};
+
+const pool = new Pool({
+    // what do we put here? do we have a db url?
```

Comment:
Not sure what to have here

Reply:
This is a good example on what you can do. https://riptutorial.com/node-js/example/4587/using-a-connection-pool
If you look in the article it exports a pool connection so we can use that and carry out the query. It seems that you have a `db-helpers.js` file that does exactly that. Technically you can remove this portion and have it as is(with your db-helpers.js fie).

Reply:
Word, thanks. Just to clarify, do you mean just straight delete this file and contents? Or move the contents of this file into `db-helpers.js`, then delete?

And the server runs but as far as testing goes I haven't been testing the queries yet (like I just mentioned in slack since my local db isn't up yet)

Reply:
Right so you can delete lines 11-13, and just call new Pool in your db-helper.js Or instead of databaseOptions you can do:
`var pool  = mysql.createPool({
    connectionLimit : 10,
    host     : 'localhost',
    database : 'test',
    user     : 'root',
    password : 'root',
    port     : '8889'
});`

Reply:
Ahhh gotcha makes sense, thanks. Just updated